class DuplicateAccountException(ValueError):

    def __init__(self):
        super(self.__class__)